ChangeLog

```diff
@@ -2,9 +2,22 @@
 astroid's ChangeLog
 ===================
 
+
+What's New in astroid 3.1.0?
+============================
+Release date: TBA
+
+
+
+What's New in astroid 3.0.1?
+============================
+Release date: TBA
+
+
+
 What's New in astroid 3.0.0?
 =============================
-Release date: TBA
+Release date: 2023-09-25
```

Comment:
off by one, but oh well :-)

Reply:
I also blundered the maintenance branch creation, let me fix that.